PhysicsTools/NanoAOD/python/autoNANO.py

```diff
@@ -44,4 +44,6 @@ def expandNanoMapping(seqList, mapping, key):
     # Add lepton time-life info tables through customize combined with PHYS
     'LepTimeLife' : {'sequence': '@PHYS',
                      'customize': '@PHYS+PhysicsTools/NanoAOD/leptonTimeLifeInfo_common_cff.addTimeLifeInfo'},
+    # Custom BTV Nano for SF measurements or tagger training
+    'BTV' : {'customize':'PhysicsTools/NanoAOD/custom_btv_cff.BTVCustomNanoAOD'}
```

Comment:
if the BTV nano is always going to be ran with PHYS, you can add it here the same way as for LepTimeLife above, and change in relval_nano accordingly.

Reply:
modified to `'@PHYS+PhysicsTools/NanoAOD/custom_btv_cff.BTVCustomNanoAOD`